classes/analytics/analytics_for_recordings_table.php

```diff
@@ -94,7 +100,10 @@ public function col_status($row) {
 
     public function col_createtime($row) {
         $data = json_decode($row->meta);
-        return userdate(($data->createtime ?? $row->timecreated) / 1000, $this->datetimeformat);
+        $time = !empty($data->createtime) ? $data->createtime / 1000 : $row->timecreated;
+        // Store time in class to use in filtering.
+        $this->time = $time;
```

Comment:
Interesting, I did not know you can set a value against the object and use it later on in the build_table function and have it correspond to the specific row. I would have thought it would have been linked against the form itself.

Reply:
yeah its a neat trick to grab computed values that you need later, rather than trying to string match against the formatted return